backend/app/views/spree/admin/shared/preference_fields/_boolean.html.erb

```diff
@@ -0,0 +1,13 @@
+<% label = local_assigns[:label].presence %>
+<% html_options = local_assigns[:html_options] || {} %>
+
+<div class="field">
+  <% if local_assigns[:form] %>
+    <%= form.check_box attribute, html_options %>
+    <%= form.label attribute, label %>
+  <% else %>
+    <%= hidden_field_tag name, 0 %>
```

Comment:
I wonder if we can simplify these cases using the new [`form_with`](https://github.com/rails/rails/issues/25197) :thinking: